train_dolly.py

```diff
@@ -158,31 +178,34 @@
 
 # COMMAND ----------
 
-# MAGIC !deepspeed {num_gpus_flag} \
-# MAGIC     --module training.trainer \
-# MAGIC     --input-model {input_model} \
-# MAGIC     --deepspeed {deepspeed_config} \
-# MAGIC     --epochs 2 \
-# MAGIC     --local-output-dir {local_output_dir} \
-# MAGIC     --dbfs-output-dir {dbfs_output_dir} \
-# MAGIC     --per-device-train-batch-size 6 \
-# MAGIC     --per-device-eval-batch-size 6 \
-# MAGIC     --logging-steps 10 \
-# MAGIC     --save-steps 200 \
-# MAGIC     --save-total-limit 20 \
-# MAGIC     --eval-steps 50 \
-# MAGIC     --warmup-steps 50 \
-# MAGIC     --test-size 200 \
-# MAGIC     --lr 5e-6
+!deepspeed {num_gpus_flag} \
```

Comment:
I don't know if this can be un-commented; this is meant to be read as a notebook.

Reply:
Strange... I don't recall making this change. I'll revert it.

Reply:
For these `!` execution cells, the `# MAGIC` comments doesn't seem to be necessary. See the `!wget` command on lines 37-44. In any event, I didn't intentionally change this - it's just how the notebook was committed after some editing in my DB workspaces.

Reply:
ok - we should probably similarly comment the `!wget` command, too.